### DISCUSSION TITLE
Release cycle of NSI

### DISCUSSION THREAD

**Discussion by Dimitar5555** [osmlab/name-suggestion-index#10205] [General]
Since Jan/Feb 2024 NSI is on a biweekly release cycle instead of the weekly release cycle previously used. Was there any particular reason for this change? 

https://github.com/osmlab/name-suggestion-index/tags?after=v6.0.20240310

**Comment by LaoshuBaby** [comment]
<img src="https://github.com/user-attachments/assets/ebd7e311-9df7-4fbc-91c5-6b4dc10b4d85" height=500px/>


In fact not exactly biweekly, but irregular, I guess is because of maintainer's lack of time? (I'd like to help if we need weekly publish, but I think it is not appropriate if I have npm permissions 🥺)

**Reply by Dimitar5555** [reply]
I was thinking of automating it but it seems to require a bit more than .yaml file (according to https://github.com/osmlab/name-suggestion-index/issues/5121).

**Reply by Snowysauce** [reply]
I started publishing releases with #9992 after the NSI went around three weeks without an update in September. I'm new enough as a regular contributor/maintainer that I didn't know that the release schedule was weekly at one point, and I'd be happy to put out a new release every weekend.

I'm assuming that most of the longer tenured maintainers have had busy schedules over the past few months. I've taken on more tasks in the interim in their absence - not as a power grab, but just to kind of keep the ship afloat. Hopefully I haven't ruffled too many feathers along the way, and that my efforts to help the project have actually been helpful.
